src/getLocationForJsonPath.ts

```diff
@@ -66,16 +70,22 @@ function getEndPosition(node: YAMLNode): number {
   return node.endPosition;
 }
 
-function findNodeAtPath(node: YAMLNode, path: JsonPath, closest: boolean) {
+function findNodeAtPath(
+  node: YAMLNode,
+  path: JsonPath,
+  { closest, mergeKeys }: { closest: boolean; mergeKeys: boolean }
```

Comment:
Isn't this a the same type as `metadata`? Maybe you could reuse and `Pick`?

Reply:
It's not.